docs_src/dslx_std.md

```diff
@@ -15,20 +15,21 @@ in a user-defined function, and thus do not live in the standard library.
 This section describes the built-in functions provided for use in the DSL that
 do not need to be explicitly imported.
 
-Some of the built-ins have exclamation marks at the end; e.g. `assert!` --
+Some of the built-ins have exclamation marks at the end; e.g., `assert!` --
 exclamation marks in DSLX indicate built-in functions that have special
 facilities that normal functions are not capable of. This is intended to help
 the user discern when they're calling "something that will behave like a normal
 function" versus "something that has special abilities beyond normal functions,
-e.g. special side-effects".
+e.g., special side-effects".
 
 !!! NOTE
     A brief note on "Parallel Primitives": the DSL is expected to grow
```

Comment:
I think we might also want to s/the DSL/DSLX/ here and below for consistency, but that can also be done as a follow-up.